packages/api/src/schema.ts

```diff
@@ -1912,6 +1912,93 @@ const schema = gql`
     BAD_REQUEST
   }
 
+  union RulesResult = RulesSuccess | RulesError
+
+  type RulesSuccess {
+    rules: [Rule!]!
+  }
+
+  type Rule {
+    id: ID!
+    name: String
+    query: String!
+    actions: [RuleAction!]!
+    triggerType: RuleTriggerType!
+    createdAt: Date!
+    updatedAt: Date!
+  }
```

Comment:
@jacksonh `query` is the search filter and `actions` referred to a list of "actions"

Reply:
For RuleTriggerType, would the client have to set that when creating a rule?

Reply:
Probably no. The `RuleTriggerType` should bind to the `action` so I probably should have a mapping in the backend and remove this from the GQL request

Reply:
Yeah I guess we'd have a list and it would be part of it.